packages/manager/modules/vps/src/upgrade/vps-upgrade.controller.js

```diff
@@ -258,7 +258,7 @@ export default class VpsUpgradeCtrl {
                 name: this.stateVps.model.offer,
                 details: VpsUpgradeCtrl.findMatchingUpgradeOffer(
                   this.stateVps.offerType,
-                  this.stateVps.model.name,
+                  this.a.model.name,
```

Comment:
I suspect a typo. I think we can keep it as:

```suggestion
                  this.stateVps.model.name,
```

Reply:
Yes, thank you, fixing it